plugin/transfer/transfer_test.go

```diff
@@ -276,3 +283,36 @@ func TestTransferNotAllowed(t *testing.T) {
 		t.Errorf("Expected REFUSED response code, got %s", dns.RcodeToString[w.Msg.Rcode])
 	}
 }
+
+type badwriter struct {
```

Comment:
I would rather see a new test which shows we are erroring

Reply:
Change the test code to see if a message write error was returned.

Reply:
please revert changes to this test and add a new one